app/services/npq/funding_eligibility.rb

```diff
@@ -2,10 +2,13 @@
 
 module NPQ
   class FundingEligibility
-    attr_reader :trn, :npq_course_identifier
+    attr_reader :npq_course_identifier,
+                :trn,
+                :get_an_identity_id
 
-    def initialize(trn:, npq_course_identifier:)
+    def initialize(npq_course_identifier:, trn: nil, get_an_identity_id: nil)
```

Comment:
This addition of a default value for trn does introduce a change where using one isn't strictly necessary. At first glance this may seem like it would impact the existing API to make it no longer require a TRN. Howver, the TRN in that API is in the URL, if you don't send one you'll get a routing error.